submariner/OVN-Interconnect.md

```diff
@@ -0,0 +1,126 @@
+# Submariner Enhancement for OVN Interconnect
+
+<!-- Add link to issue/epic if available -->
+
+## Summary
+
+OVN CNI is moving to a new network topology powered by OVN Interconnect Feature. This allows independent
+OVN deployments to be interconnected by OVN-managed GENEVE tunnels. Submariner does not work with these changes
```

Comment:
Can you please link relevant OVN issues/PRs here for reference?

Reply:
Added in external dependencies.